src/java.base/share/classes/java/io/Reader.java

```diff
@@ -140,6 +140,208 @@ public void close() {
         };
     }
 
+    /**
+     * Returns a new {@code Reader} whose source is a {@link CharSequence}.
+     *
+     * <p> The returned reader's class provides an API compatible with
+     * {@link StringReader}, but with no guarantee of synchronization, and
+     * without limitation to {@code String} sources. This class is designed for
+     * use as a drop-in replacement for {@code StringReader} in places where the
+     * reader was being used by a single thread (as is generally the case).
+     * Where possible, it is recommended that this class be used in preference
+     * to {@code StringReader} as it will be faster under most implementations.
```

Comment:
I believe this paragraph is redundant, now that `StringReader` is a somewhat-deprecated API that we aim to migrate away from. We can note these differences in the API notes of `StringReader`.

Reply:
As Alan already proposed, I am just removing it in this very minute.